tests/beaker/tests/aix/create_lv_with_param_min_range_aix.rb

```diff
@@ -0,0 +1,49 @@
+require 'master_manipulator'
+require 'lvm_helper'
+require 'securerandom'
+
+test_name "FM-4969 - C97372 - create logical volume with parameter min 'range' aix"
```

Comment:
Incorrect TestRail ID specified and test name does not match. I think you mixed this up with C97367.